internal/network/cidr_validations.go

```diff
@@ -59,6 +59,13 @@ func max(x, y int) int {
 	return y
 }
 
+func min(x, y int) int {
+	if x < y {
+		return x
+	}
+	return y
+}
+
 func VerifyClusterCidrSize(hostNetworkPrefix int, clusterNetworkCIDR string, numberOfHosts int) error {
```

Comment:
I couldn't find UT for this function. Probably they're worth adding.

Reply:
They are part of validations (transitions_test).  I will add some tests